nixos/modules/installer/cd-dvd/iso-image.nix

```diff
@@ -50,7 +50,7 @@ let
     finalCfg = {
       name = "NixOS ${config.system.nixos.label}${config.isoImage.appendToMenuLabel}";
       params = "init=${config.system.build.toplevel}/init ${additional} ${toString config.boot.kernelParams}";
-      image = "/boot/bzImage";
+      image = "/boot/${config.system.boot.loader.kernelFile}";
```

Comment:
IIRC this can always be `hostPlatform.kernelTarget` or something (`system.boot.loader.kernelFile` should be a totally redundant option).

Reply:
I'd prefer not to change it, I'm not too keen on doing a complete refactor while the rest of the file is already using that idiom. Though I agree will be better to use `hostPlatform.kernelTarget` where it is defined later in the file, instead of the conditional I used, but by removing the line and relying on its default value (which is `pkgs.stdenv.hostPlatform.platform.kernelTarget;`).

Reply:
Makes sense to me :+1:.